Eatery/Onboarding/Controllers/OnboardingLoginViewController.swift

```diff
@@ -0,0 +1,254 @@
+    private var netidTextField: UITextField!
+
+    private var passwordPrompt: UILabel!
+    private var passwordTextField: UITextField!
+
+    private var loginButton: UIButton!
+    private var skipButton: UIButton!
+    private var privacyStatementButton: UIButton!
+
+    private var activityIndicator: NVActivityIndicatorView!
+    
+    private var errorView: BRBLoginErrorView!
+    var errorDescription: String? {
+        get { return errorView.errorLabel.text }
+        set { errorView.errorLabel.text = newValue }
+    }
+
+    let accountManager = BRBAccountManager()
+
+    private var isLoading: Bool = false {
+        didSet {
+            if isLoading {
+                activityIndicator.startAnimating()
+                loginButton.setTitle(nil, for: .normal)
+            } else {
+                activityIndicator.stopAnimating()
+                loginButton.setTitle("LOG IN", for: .normal)
+            }
+
+            stackView.isUserInteractionEnabled = !isLoading
+            loginButton.isEnabled = !isLoading
+        }
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        accountManager.delegate = self
+
+        loginStackView = UIStackView(frame: .zero)
+        loginStackView.axis = .vertical
+        loginStackView.distribution = .fill
+        loginStackView.spacing = 10
+        stackView.insertArrangedSubview(loginStackView, at: 2)
+        stackView.setCustomSpacing(20, after: subtitleLabel)
+        stackView.setCustomSpacing(40, after: loginStackView)
+        loginStackView.snp.makeConstraints { make in
+            make.width.equalToSuperview()
+        }
+
+        setUpPrivacyStatementButton()
+        setUpErrorView()
+        setUpNetidViews()
+        setUpPasswordViews()
+        setUpSkipButton()
+    }
+
+    private func setUpPrivacyStatementButton() {
+        privacyStatementButton = UIButton(type: .system)
+        privacyStatementButton.setTitle("Privacy Statement", for: .normal)
+        privacyStatementButton.setTitleColor(.white, for: .normal)
+        privacyStatementButton.titleLabel?.font = .preferredFont(forTextStyle: .subheadline)
+        privacyStatementButton.setTitleColor(.black, for: .highlighted)
+        privacyStatementButton.addTarget(self,
+                                         action: #selector(privacyStatementButtonPressed(_:)),
+                                         for: .touchUpInside)
+        loginStackView.addArrangedSubview(privacyStatementButton)
+    }
+
+    @objc private func privacyStatementButtonPressed(_ sender: UIButton) {
+        let privacyStatementViewController = BRBPrivacyStatementViewController()
+        self.present(privacyStatementViewController, animated: true, completion: nil)
+    }
+
+    private func setUpErrorView() {
+        errorView = BRBLoginErrorView(frame: .zero)
+        errorView.isCollapsed = true
+        loginStackView.addArrangedSubview(errorView)
+    }
+
+    private func setUpNetidViews() {
```

Comment:
In the rest of the app did you use `setup` or `setUp`? Either way, just be consistent.

Reply:
lets use `setup`

Reply:
Rest of the app uses `setUp` (or at least should), `setup` is a noun

Reply:
I used setUp since that's what was used globally throughout the app